### PR TITLE
Sds field  name value pair

### PR DESCRIPTION
**Pull request by AlejandroColinaGSA**

### CODE REVIEW

File: src/components/01-branding/02-typography/fields.njk [hunk 1]
```diff
@@ -1,69 +1,67 @@
...
-
 
 <h2>Inline Field</h2>
 <p>Display name and value inline.</p>
 <div class="grid-row">
     <div class="sds-field">
-        <div class="sds-name">Field:</div>
-        <div class="sds-value">Lorem ipsum dolor</div>
+        <div class="sds-field--name">Field:</div>
+        <div class="sds-field--value">Lorem ipsum dolor</div>
```
Comment (davereed) [inline src/components/01-branding/02-typography/fields.njk]: For BEM naming, this is my take:
**Block**: sds-field
**Modifier (variant)**: sds-field--small, sds-field--featured ... (along with all the variants)
**Element (child)**: sds-field__name
**Element (child)**: sds-field__value

File: src/components/01-branding/02-typography/fields.njk [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+    </div>
+</div>
+<div class="grid-row">
+    <div class="sds-field sds-field--large">
+        <div class="sds-field__name">Field Large:</div>
+        <div class="sds-field__value">Lorem ipsum dolor</div>
+    </div>
+</div>
+<div class="grid-row">
+    <div class="sds-field sds-field--parragraph">
```
Comment (davereed) [inline src/components/01-branding/02-typography/fields.njk]: parragraph should be spelled: paragraph

File: src/components/01-branding/02-typography/fields.njk [hunk 3]
```diff
@@ -1,9 +1,71 @@
...
+        <div class="sds-field__name">Parragraph Field:</div>
+        <div class="sds-field__value">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque vehicula, mi at imperdiet imperdiet, lectus velit vehicula dolor, ac scelerisque metus lectus in eros. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.</div>
+    </div>
+</div>
+<h2>Stacked Field</h2>
+<p>Display name and value stacked.</p>
+<div class="grid-row">
+    <div class="sds-field sds-field__stacked">
```
Comment (davereed) [inline src/components/01-branding/02-typography/fields.njk]: change to sds-field--stacked (everywhere it was changed)

Comment (AlejandroColinaGSA) [reply src/components/01-branding/02-typography/fields.njk]: fixed

**Pull request merged**